### PR TITLE
Add LED support

### PR DESCRIPTION
**Pull request by davidwagnerkc**
We'll call this the pre-Derick designing crazy LED fanciness stage. 

Simple use of LEDs. 

Default - off. 

Wake word detected and listening - blue ring

Skill crunchin' some numbers and solving problems - blue/green rotating ring.

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -273,40 +276,41 @@ def shutdown(self):
...
+            return
+        pixel_ring.think()
+
     def on_handler_complete(self, message):
-        """ When a skill finishes executing clear the showing page state. """
+        """When a skill finishes executing turn off the LED ring"""
         handler = message.data.get('handler', '')
-        # Ignoring handlers from this skill and from the background clock
-        # TODO: implement Something here
+        if _skip_handler():
```
Comment (forslund) [inline __init__.py]: Same as above

File: __init__.py [hunk 2]
```diff
@@ -273,40 +276,41 @@ def shutdown(self):
...
-
     def on_handler_mouth_reset(self, message):
-        """ Restore viseme to a smile. """
+        """Restore viseme to a smile."""
         pass
 
+    def on_handler_started(self, message):
+        """When a skill begins executing turn on the LED ring"""
+        handler = message.data.get('handler', '')
+        if _skip_handler():
```
Comment (forslund) [inline __init__.py]: Should be `self._skip_handler()` maybe and there should probably be an argument as well

**Comment by davidwagnerkc** [comment]
@forslund Can you take a second look?

I added LED indication for speaking per Derick's request, added some of his initial Mycroft color palette recommendations, and made it so that when volume is set or get that instead of the `speak()` LED indicator we display the volume on the LED ring. This volume feature requires another PR to work: https://github.com/MycroftAI/skill-volume/pull/44.

**Comment by forslund** [comment]
Works fine. Merging

**Pull request merged**